components/eam/src/chemistry/mozart/lin_strat_chem.F90

```diff
@@ -311,14 +331,19 @@ subroutine linv3_strat_chem_solve(ncol, lchnk, xvmr, h2ovmr, xsfc, o3col, temp,
     if ( .not. linoz_v3) return
 
 !    write(iulog,*)'inside lin_strat_solve for ndx o3, o3lnz, n2o, noylnz, ch4', o3_ndx, o3lnz_ndx, n2o_ndx, noylnz_ndx, ch4_ndx
-
+    if (chemFlag == 'LNZ') then
         o3_vmr =  xvmr(:,:, o3lnz_ndx)
        n2o_vmr =  xvmr(:,:,n2olnz_ndx)
        noy_vmr =  xvmr(:,:,noylnz_ndx)
        ch4_vmr =  xvmr(:,:,ch4lnz_ndx)
        h2o_vmr =  xvmr(:,:,h2olnz_ndx)
-
-
+    else
+        o3_vmr =  xvmr(:,:, o3_ndx)
+       n2o_vmr =  xvmr(:,:,n2o_ndx)
```

Comment:
We don't have N2O in chemUCI, so n2o_ndx should be negative.

Reply:
Yes, I checked n2o_ndx is -1. I will use n2olnz instead of n2o.